network-api/networkapi/buyersguide/migrations/0025_bootstrap_user_friendly_privacy_policy.py

```diff
@@ -1,13 +1,24 @@
 # Generated by Django 2.2.5 on 2019-10-09 00:10
 
 from django.db import migrations, models
-from networkapi.buyersguide.models import Product
```

Comment:
this change makes sure we use a `Product` based on the database schema, _not_ what's in models.py (since models.py can change in the future, and then this migration would crash...)